lib/ActivityStream.jsm

```diff
@@ -166,7 +168,7 @@ const PREFS_CONFIG = new Map([
   }],
   ["improvesearch.topSiteSearchShortcuts", {
     title: "Experiment to show special top sites that perform keyword searches",
-    value: AppConstants.MOZ_UPDATE_CHANNEL !== "release",
+    value: UpdateUtils.getUpdateChannel() !== "release",
```

Comment:
Looks like update channel might contain some partner information (unless we pass in `false` here), is it intended that we wouldn't match those? https://searchfox.org/mozilla-central/source/toolkit/modules/UpdateUtils.jsm#

Reply:
Yeah I'm not sure about that either (see PR comment). I asked @Mardak if he had some insight on this

Reply:
Let's pass in `true` in the empty / default calls just to be explicit just in case someone thinks about adding another "release" check for something else and comes across the empty one before noticing the `false` one.